CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+
+## Upcoming - 2023-xx-xx
+
+### Fixed
+
+- `ApiTokenAuthentication` again validates the `aud` claim. The `aud` claim wasn't validated if the `drf-oidc-auth` version was 1.0.0 or greater.
+
+### Added
+- Documentation about social auth pipeline configuration
+
+### Removed 
+
+- Removed `drf-oidc-auth` requirement when using `ApiTokenAuthentication`. Django REST framework is still required.
+
+### Changed
+
+- `ApiTokenAuthentication` is no longer a subclass of `oidc_auth.authentication.JSONWebTokenAuthentication`
```

Comment:
Could add: "but a direct subclass of `rest_framework.authentication.BaseAuthentication`".

Reply:
Added.

Reply:
No you should. ☺️